src/gl/lineFeature.js

```diff
@@ -60,8 +95,15 @@ var gl_lineFeature = function (arg) {
           'uniform mat4 projectionMatrix;',
           'uniform float pixelWidth;',
           'uniform float aspect;',
+          'uniform float miterLimit;',
+          'uniform float antialiasing;',
 
           'varying vec4 strokeColorVar;',
+          'varying vec4 subpos;',  /* px, py, length - px, width */
+          'varying vec4 info;',  /* near mode, far mode, offset */
+          'varying vec4 angles;', /* near angle cos, sin, far angle cos, sin */
+
+          'const float PI = 3.14159265358979323846264;',
```

Comment:
perhaps we can define it some where else (utils?) so that we don't have to define it multiple times.